### PR TITLE
Update formal power series printing

### LINKED ISSUE #11102: fps should print as a formal power series
When I first used `fps`, I didn't realize it really was a formal power series as it claims to be, because it prints like a normal series (same as `series`)

```
In [21]: fps(sin(x))
Out[21]:
     3     5
    x     x     ⎛ 6⎞
x - ── + ─── + O⎝x ⎠
    6    120
```

But if you look at the string form, you see

```
In [22]: print(fps(sin(x)))
FormalPowerSeries(sin(x), x, 0, 1, (SeqFormula(Piecewise(((-1/4)**(_k/2 - 1/2)/(RisingFactorial(3/2, _k/2 - 1/2)*factorial(_k/2 - 1/2)), Eq(Mod(_k, 2), 1)), (0, True)), (_k, 2, oo)), SeqFormula(x**_k, (_k, 0, oo)), x))
```

That is, it really does represent it as the formula `Sum((-1)**n/factorial(2*n + 1)*x**n, (n, 0, oo))` (albiet, not simplified). It out to print it like this, so you can see that that's what it's working with.

Side question: if you enter something it can't compute, it just returns the function

```
In [25]: fps(tan(x))
Out[25]: tan(x)
```

Is that intentional? It seems like it ought to raise an exception in that case. 

@leosartaj

### PR DESCRIPTION
**Pull request by leosartaj**
fixes #11102 
ping @asmeurer

### CODE REVIEW

**Comment by leosartaj** [comment]
Can you review @sampadsaha5 ?

**Comment by sampadsaha5** [comment]
Sure. :smile:

File: sympy/printing/pretty/tests/test_pretty.py [hunk 1]
```diff
@@ -3430,20 +3430,32 @@ def test_pretty_FourierSeries():
 def test_pretty_FormalPowerSeries():
     f = fps(log(1 + x))
 
+
```
Comment (sampadsaha5) [inline sympy/printing/pretty/tests/test_pretty.py]: Please remove this blank line.

File: sympy/printing/pretty/tests/test_pretty.py [hunk 2]
```diff
@@ -3430,20 +3430,32 @@ def test_pretty_FourierSeries():
...
+  ∞              \n\
+ ____            \n\
+ ╲               \n\
+  ╲        -k  k \n\
+   ╲  -(-1)  ⋅x  \n\
+   ╱  ───────────\n\
+  ╱        k     \n\
+ ╱               \n\
+ ‾‾‾‾            \n\
+k = 1            \
```
Comment (sampadsaha5) [inline sympy/printing/pretty/tests/test_pretty.py]: How about printing this way ?
```py  
 oo             -k    k  
 __        -(-1)  ⋅ x 
\           -------------- 
/__               k
k = 1       
```

Comment (asmeurer) [inline sympy/printing/pretty/tests/test_pretty.py]: This is just reusing the `Sum` printer, so if you want to change that you should open a new issue.

**Pull request merged**